client/components/homepage/Features.jsx

```diff
@@ -6,7 +6,7 @@ const Features = () => (
       <div className="container">
         <div className="row text-center-mobile">
           <div className="col-md-6">
-            <div className="icon"><i className="pe-7s-diamond" /></div>
+            <div className="icon"><i className="glyphicon glyphicon-list-alt" /></div>
```

Comment:
Great job with adding the glyphicons - I'd like to take a look at how to center them within the circle though. Maybe @dshelters can also have a look at this while he restyles other aspect of the landing page.